gnosis/safe-react
# [L2-UX] Merge the selected currency across networks into one localStorage key

## What it solves
Resolves #2728

- Used USD as a default currency if no selected currency is present in the localStorage.
- `SAFE__currencyValues.selectedCurrency` key used to store the selected currency in the localStorage.
- Added unit tests to `CurrencyDropdown` component
- We use the same selected currency value across all networks.

## How to test it

1. Go to `{{baseUrl}}/app/rinkeby/safes/{{safeAddress}}/balances`
2. We use USD as a default value
3. Change the selected currency in the dropdown
4. Check the new selected currency value in the localStorage (`SAFE__currencyValues.selectedCurrency` key)
5. Refresh the page and see the custom selected currency value in the Dropdown
6. We use the same selected currency value across all networks.

## Screenshots
![Captura de pantalla 2021-10-05 a las 11 12 18](https://user-images.githubusercontent.com/26763673/135994974-77a367cf-9e39-4867-abe5-f727b0dfc99e.png)
